### ISSUE TITLE
How to copydoc documentation of functions with spaces?

### ISSUE THREAD

**Issue by Xeverous** [doxygen/doxygen#11169]
**Describe the bug**
On https://www.doxygen.nl/manual/commands.html#cmdcopydoc, it is described that if one wants to copy a documentation of specific overload: "should specify the argument types explicitly (without spaces!)" `//! @copydoc MyClass::myfunction(type1,type2)`

How am I supposed to `@copydoc` an overload that takes `unsigned long` then? Is this not supported?

**To Reproduce**
In my specific case, I have a code that looks like this:

```cpp
/**
 * @brief X
 */
struct X
{
	/**
	 * @brief desc
	 * @return true
	 */
	explicit operator bool() const &
	{
		return true;
	}
};

/**
 * @brief Y
 */
struct Y
{
	/**
	 * @copydoc X::operator bool() const &
	 */
	explicit operator bool() const &
	{
		return true;
	}
};
```

and it generated the following error: `warning: @copybrief or @copydoc target 'X::operator' not found`. Looks like Doxygen stops on the "operator" word and does not parse further. Is this intended?

**Expected behavior**
Support for complex overload sets with const and reference qualifiers, including convertion operators.

**Version**
Doxygen: `1.9.8 (c2fe5c3e4986974eb2a97608b24086683502f07f)`
OS:
```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.6 LTS
Release:	20.04
Codename:	focal
```
Binary taken from the website.

**Comment by Xeverous** [comment]
- `unsigned long` - this is only a hypothetical question. I haven't tested it but it conflicts with documentation recommendation that overloads should not have spaces.

Will try to reproduce with newer version soon.

**Comment by albert-github** [comment]
> `unsigned long` - this is only a hypothetical question. I haven't tested it but it conflicts with documentation recommendation that overloads should not have spaces.

As far as I know this is not meant for spaces inside types but for spaces around the type (so after the `(` or `,` and before the `)` or `,`), could be explained a bit more explicit though.

**Comment by Xeverous** [comment]
No reproduction, can not launch newest version:
```
doxygen: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by doxygen)
doxygen: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by doxygen)
doxygen: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by doxygen)
doxygen: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by doxygen)
doxygen: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by doxygen)
doxygen: /lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.13' not found (required by doxygen)
```

**Comment by albert-github** [comment]
The glibc you are using looks like not to be correct looks like similar problem as noted in #10943

**Comment by Xeverous** [comment]
Yes, this is the same problem. A suddenly high requirement of libc and libcxx.

**Comment by Xeverous** [comment]
[repro.zip](https://github.com/user-attachments/files/17247829/repro.zip)

Best I can do is version 1.10, same result.

**Comment by albert-github** [comment]
@Xeverous 

Regarding your version problem (not important for the `copydoc` problem as I can reproduce this with the given example)
- where did you get the doxygen 1.12.0 version from? 
   - github, doxygen.org/download  they should work as they are compiled on Ubuntu 20.04 (and gcc 9.4.0, at least on github)
   - Ubuntu repository? 
   - somewhere else?

**Comment by Xeverous** [comment]
I downloaded the binary from Doxygen's website and used its make install script.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
